flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/maintenance/api/MaintenanceTaskBuilder.java

```diff
@@ -41,6 +42,26 @@ public abstract class MaintenanceTaskBuilder<T extends MaintenanceTaskBuilder<?>
 
   abstract DataStream<TaskResult> append(DataStream<Trigger> sourceStream);
 
+  @VisibleForTesting
+  Integer getScheduleCommitCount() {
```

Comment:
Please check the https://iceberg.apache.org/contribute/#java-style-guidelines
In the Iceberg code base we omit things like `get`